docs/developer-docs/latest/developer-resources/cli/snippets/installation-prerequisites.md

```diff
@@ -1,5 +1,20 @@
-The installation requires the following software to be already installed on your computer:
+Before installing Strapi, the following requirements must be installed on your computer:
 
-- [Node.js](https://nodejs.org): only maintenance and LTS versions are supported (v14, v16, and v18). Other versions of Node.js may not be compatible with the latest release of Strapi. Node v18.x is recommended for Strapi v4.3.9 and above, Node v16.x for Strapi v4.0.x to v4.3.8.
-- [npm](https://docs.npmjs.com/cli/v6/commands/npm-install) (v6 only) or [yarn](https://yarnpkg.com/getting-started/install) to run the CLI installation scripts.
-- [Python](https://www.python.org/downloads/) when using a SQLite database
+- [Node.js](https://nodejs.org): Only Maintenance and LTS versions are supported (`v14`, `v16`, and `v18`).
```

Comment:
This snippet is also used in the Quick Start Guide.
I'm not sure we should have such a long prerequisites block in the QSG and would recommend you only update it for the CLI installation page. 🤔 Or maybe we could use a toggle (`::: details` blog) in the QSG. What do you think?